README.md

```diff
@@ -10,11 +10,22 @@
 
 LitElement uses [lit-html](https://github.com/Polymer/lit-html) to render into the
 element's [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM)
-and [Polymer's](https://github.com/Polymer/polymer)
-[PropertiesMixin](https://github.com/Polymer/polymer/blob/master/lib/mixins/properties-mixin.js)
-to help manage element properties and attributes. LitElement reacts to changes in properties
+and adds API help manage element properties and attributes. LitElement reacts to changes in properties
 and renders declaratively using `lit-html`.
 
+  * **Setup properties:** LitElement supports a static `properties` getter in which element property
+  accessors are described. For each property, an object configures settings where the options are:
+
+    * attribute: if false, do not add to observedAttributes, if true or absent, observe the
```

Comment:
Instead of doing the camel casing automatically, why not also allow `attribute: my-custom-name` as well, so that I can choose the mapping name that I want

Reply:
This seems to be exactly what you are doing in the TS example:

```@property({attribute: 'more-info', type: (value: string) => `[${value}]`})```

Reply:
OK, you seems to be doing exactly what I wanted, but the documentation doesn't mention that

Reply:
Done